### PR TITLE
Set up opseng-challenge repo

### PR DESCRIPTION
**Pull request by mwarkentin**
- [x] Verify timing of challenge
- [x] Determine if we're reusing the `dev.careers@waveapps.com` email
- [x] Update challenge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,69 @@
-# opseng-challenge
-Wave OpsEng Development Challenge
+# Wave OpsEng Development Challenge
+
+Applicants for the [Operations Engineer career](https://www.waveapps.com/about-us/jobs/operations-engineer/) at Wave must complete the following challenge, and submit a solution prior to the interviewing process. This will help the interviewers assess your strengths, and frame the conversation through the interview process. Take as much time as you need, however we ask that you not spend more than a few hours.
+
+Send your submission to [dev.careers@waveapps.com](dev.careers@waveapps.com). Feel free to email [dev.careers@waveapps.com](dev.careers@waveapps.com) if you have any questions.
```
Comment (mwarkentin) [inline README.md]: @nduthoit Are we using the same email?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,69 @@
...
+
+## Submission Instructions
+
+1. Fork this project on github. You will need to create an account if you don't already have one
```
Comment (mwarkentin) [inline README.md:9]: @nduthoit The challenge we currently have is composed of two separate repos (one for the flask app, one for the provisioning code) - this doesn't really make sense with this instruction. Ok to modify this to reference creating your own repos, or did we want to try to update the challenge to work out of a single forked repo?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,69 @@
-# opseng-challenge
-Wave OpsEng Development Challenge
+# Wave OpsEng Development Challenge
+
+Applicants for the [Operations Engineer career](https://www.waveapps.com/about-us/jobs/operations-engineer/) at Wave must complete the following challenge, and submit a solution prior to the interviewing process. This will help the interviewers assess your strengths, and frame the conversation through the interview process. Take as much time as you need, however we ask that you not spend more than a few hours.
```
Comment (mwarkentin) [inline README.md]: > prior to the interviewing process.

@nduthoit Is this the case for us? I believe previously when we discussed this, we talked about doing a phone screen first, and then asking them to complete the code challenge.

Comment (nduthoit) [inline README.md]: The phone screen may happen before or after. I think the language is fine the way it is now.

We should tweak the time taken to do it though. How long did it take you?

Comment (mwarkentin) [inline README.md]: Ok. Hard to say, it was spread out over a few days.. I'd say probably 4-6 hours.

Comment (nduthoit) [inline README.md]: Ok, can you adjust it to say 4-8 hours?

**Comment by nduthoit** [comment]
LGTM

**Pull request merged**